pkg/resources/standard_database.go

```diff
@@ -0,0 +1,701 @@
+	),
+	"storage_serialization_policy": nestedProperty(
+		schema.TypeString,
+		fmt.Sprintf("Specifies the storage serialization policy for Iceberg tables that use Snowflake as the catalog. Valid options are: %v. COMPATIBLE: Snowflake performs encoding and compression of data files that ensures interoperability with third-party compute engines. OPTIMIZED: Snowflake performs encoding and compression of data files that ensures the best table performance within Snowflake.", sdk.AsStringList(sdk.AllStorageSerializationPolicies)),
+	),
+	"log_level": nestedPropertyWithInnerModifier(
+		schema.TypeString,
+		fmt.Sprintf("Specifies the severity level of messages that should be ingested and made available in the active event table. Valid options are: %v. Messages at the specified level (and at more severe levels) are ingested. For more information, see [LOG_LEVEL](https://docs.snowflake.com/en/sql-reference/parameters.html#label-log-level).", sdk.AsStringList(sdk.AllLogLevels)),
+		func(inner *schema.Schema) {
+			inner.DiffSuppressFunc = func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+				return strings.EqualFold(oldValue, newValue) || (d.Get(k).(string) == string(sdk.LogLevelOff) && newValue == "")
+			}
+		},
+	),
+	"trace_level": nestedProperty(
+		schema.TypeString,
+		fmt.Sprintf("Controls how trace events are ingested into the event table. Valid options are: %v. For information about levels, see [TRACE_LEVEL](https://docs.snowflake.com/en/sql-reference/parameters.html#label-trace-level).", sdk.AsStringList(sdk.AllTraceLevels)),
+	),
+	"replication": {
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "Configures replication for a given database. When specified, this database will be promoted to serve as a primary database for replication. A primary database can be replicated in one or more accounts, allowing users in those accounts to query objects in each secondary (i.e. replica) database.",
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"enable_for_account": {
+					Type:        schema.TypeList,
+					Optional:    true,
+					Description: "Entry to enable replication and optionally failover for a given account identifier.",
+					MinItems:    1,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"account_identifier": {
+								Type:     schema.TypeString,
+								Required: true,
+								// TODO(SNOW-1438810): Add account identifier validator
+								Description: "Specifies account identifier for which replication should be enabled. The account identifiers should be in the form of `\"<organization_name>\".\"<account_name>\"`.",
+							},
+							"with_failover": {
```

Comment:
I joined them, because failover is not possible without replication enabled, so I added failover as optional value that can be added for enabling replication.